website/components/search/style.css

```diff
@@ -0,0 +1,369 @@
+:root {
```

Comment:
This is a prime candidate for converting to css modules, especially with the injection of all these global variables that don't need to be global